client/src/components/loadingChart.tsx

```diff
@@ -5,7 +5,7 @@ import LoadingEllipsis from './loadingEllipsis';
 export default function Chart() {
     return (
         <div className='charts_donut'>
-            <Donut />
+            <Donut percent={0} percent2={0} />
```

Comment:
I believe one of the recent updates made `percent` and `percent2` optional, so I don't think this change is needed anymore

Reply:
I saw that.  Going to close this MR and get the new content merged in